bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BigtableAsyncRpc.java

```diff
@@ -43,14 +44,14 @@
 
     public static RpcMetrics createRpcMetrics(MethodDescriptor<?, ?> descriptor) {
       String fullMethodName = descriptor.getFullMethodName();
-      MetricRegistry registry = BigtableClientMetrics.getMetricRegistry();
+      MetricRegistry debugRegistry = getMetricRegistry(MetricLevel.Debug);
+      MetricRegistry infoRegistry = getMetricRegistry(MetricLevel.Info);
       return new RpcMetrics(
-          registry.createTimer(fullMethodName + ".operation.latency"),
-          registry.createTimer(fullMethodName + ".rpc.latency"),
-          registry.createCounter(fullMethodName + ".retry.count"),
-          registry.createCounter(fullMethodName + ".failure.count"),
-          registry.createCounter(fullMethodName + ".retries.exhausted.count")
-          );
+          infoRegistry.createTimer(fullMethodName + ".operation.latency"),
```

Comment:
Suggested this out-of-band to @sduskis but one nice way to handle this instead would be to have our own createTimer, createCounter methods that take the metric string name and return the appropriate registry. 

For example we could append ".loglevel" to the end of the strings and then look them up in the slf4j config.